doc/source/ray-air/package-ref.rst

```diff
@@ -167,6 +167,12 @@ Scikit-Learn
     :members:
     :show-inheritance:
 
+FunctionTrainer
```

Comment:
can we add a section of the docs to talk about how to use the scaling config if you want to specify more resources?

Reply:
There is an example in the docstring. I'm wondering if this should just be updated here https://docs.ray.io/en/master/tune/tutorials/tune-resources.html when we make the switch to document the AIR API instead of tune.run.

Otherwise, can you point me to where a good place would be to put this? I was thinking a FAQ section makes most sense for this.